libraries/standard/mqtt/src/mqtt.c

```diff
@@ -41,6 +41,14 @@
     #define MQTT_MAX_CONNACK_RECEIVE_RETRY_COUNT    ( 5U )
 #endif
 
+/**
+ * @brief Number of milliseconds to wait for a ping response.
```

Comment:
Would be goof to mention that it is related to keep-alive in the comment.
```suggestion
 * @brief Number of milliseconds to wait for a ping response packet when a ping request packet is sent.
```

Reply:
Changed